app/screens/BookDetail/layout.js

```diff
@@ -1,15 +1,29 @@
 import React from 'react';
-import { View } from 'react-native';
+import PropTypes from 'prop-types';
+import { Text, View, FlatList } from 'react-native';
+
+import { bookDataPropType, commentDataPropType } from '../../../propTypes';
 
 import BookDetailOptions from './components/BookDetailOptions';
 import { styles } from './styles';
 
-function BookDetail(props) {
+function BookDetail({ bookData, commentsData, renderComment, keyExtractor }) {
   return (
     <View style={styles.container}>
-      <BookDetailOptions {...props} />
+      <BookDetailOptions data={bookData} />
+      <View style={styles.commentsContainer}>
+        <FlatList data={commentsData} renderItem={renderComment} keyExtractor={keyExtractor} />
+        <Text style={styles.viewAllText}>View All</Text>
```

Comment:
since this is supposed to be a button keep in mind that it should be a `TouchableOpacity`, leave it as it is since it's not going to be functional